### PR TITLE
Sync itrex1.3

### PR DESCRIPTION
**Pull request by airMeng**
## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description

detail description 
Issues: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: bestla/jblas/kernel_avx512f.h [hunk 1]
```diff
@@ -888,10 +903,10 @@ inline void f32_f4_quantize_4x16(const float* srcptr, int8_t* dstptr, int ld_src
...
     zmm3 = _mm512_add_ps(zmm3, zmm_zp);
   } else {
-    mask4 = _mm512_cmplt_ps_mask(zmm0, zmm_v0);
-    mask5 = _mm512_cmplt_ps_mask(zmm1, zmm_v0);
-    mask6 = _mm512_cmplt_ps_mask(zmm2, zmm_v0);
-    mask7 = _mm512_cmplt_ps_mask(zmm3, zmm_v0);
+    mask4 = _mm512_cmp_ps_mask(zmm0, zmm_v0, 1);
+    mask5 = _mm512_cmp_ps_mask(zmm1, zmm_v0, 2);
+    mask6 = _mm512_cmp_ps_mask(zmm2, zmm_v0, 3);
+    mask7 = _mm512_cmp_ps_mask(zmm3, zmm_v0, 4);
```
Comment (zhewang1-intc) [inline bestla/jblas/kernel_avx512f.h:909]: why there are 4 different imm8?

Comment (airMeng) [reply bestla/jblas/kernel_avx512f.h]: fixed

**Review by VincyZhang — APPROVED** [review]

File: CMakeLists.txt [hunk 2]
```diff
@@ -59,11 +59,7 @@ option(NE_AVX512_VBMI            "neural_engine: enable AVX512-VBMI"
 option(NE_AVX512_VNNI            "neural_engine: enable AVX512-VNNI"                             OFF)
 option(NE_FMA                    "neural_engine: enable FMA"                                     ON)
 option(NE_AMX                    "neural_engine: enable AMX"                                     OFF)
-
-# in MSVC F16C is implied with AVX2/AVX512
-if (NOT MSVC)
-    option(NE_F16C               "neural_engine: enable F16C"                                    ON)
-endif()
+option(NE_F16C               "neural_engine: enable F16C"                                        ON)
```
Comment (zhewang1-intc) [inline CMakeLists.txt:62]: quite confused about neural_engine here, it should already be deprecated.

File: CMakeLists.txt [hunk 3]
```diff
@@ -59,11 +59,7 @@ option(NE_AVX512_VBMI            "neural_engine: enable AVX512-VBMI"
...
 
 # 3rd party libs
 option(NE_ONEDNN                 "neural_engine: use oneDNN"                                     ON)
```
Comment (zhewang1-intc) [inline CMakeLists.txt:65]: consider to clean the cmakelist?

**Review by zhewang1-intc — APPROVED** [review]

File: CMakeLists.txt (hunk 2 above)
Comment (airMeng) [reply CMakeLists.txt:62]: naming in code will be updated after 0.1

File: CMakeLists.txt (hunk 3 above)
Comment (airMeng) [reply CMakeLists.txt:65]: same

**Pull request merged**